superbench/benchmarks/model_benchmarks/model_base.py

```diff
@@ -400,8 +400,8 @@ def __process_model_result(self, model_action, precision, step_times):
         # The unit of step time is millisecond, use it to calculate the throughput with the unit samples/sec.
```

Comment:
Would you please add new parameters in document as well? I think it should be in API.

Reply:
Thanks, have updated the docs.